package.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
This file should not be in the repository -- only the `package.json` file.  `npm` generates this file automatically.